pkg/minishift/network/settings.go

```diff
@@ -43,6 +58,106 @@ type NetworkSettings struct {
 	Gateway   string
 	DNS1      string
 	DNS2      string
+	UseDHCP   bool
+	Disabled  bool
+}
+
+func checkSupportForAddressAssignment() bool {
+	if minishiftConfig.InstanceConfig.IsRHELBased &&
+		minishiftConfig.InstanceConfig.SupportsNetworkAssignment {
+		return true
+	} else {
+		atexit.ExitWithMessage(1, configureNetworkNotSupportedMessage)
+	}
+	return false
+}
+
+// This will return the address as used by libmachine or assigned by us
+func GetIP(driver drivers.Driver) (string, error) {
+	/*
+		if HasStaticIP() {
+			return GetStaticIP(), nil
+		}
+	*/
+
+	ip, err := driver.GetIP()
+	if err != nil {
+		return "", err
+	}
+	return ip, nil
+}
+
+func ConfigureDynamicAssignment(driver drivers.Driver) {
+	if checkSupportForAddressAssignment() {
+		fmt.Println("Writing configuration for dynamic assignment of IP address")
+	}
+
+	networkSettingsEth0 := NetworkSettings{
+		Device:  "eth0",
+		UseDHCP: true,
+	}
+	WriteNetworkSettingsToInstance(driver, networkSettingsEth0)
+
+	networkSettingsEth1 := NetworkSettings{
+		Device:  "eth1",
+		UseDHCP: true,
+	}
+	WriteNetworkSettingsToInstance(driver, networkSettingsEth1)
+
+	//minishiftConfig.InstanceConfig.IPAddress = ""
+	//minishiftConfig.InstanceConfig.Write()
+
+	fmt.Println(configureRestartNeededMessage)
+}
+
+func ConfigureStaticAssignment(driver drivers.Driver) {
+	// Not supported for KVM
+	if minishiftConfig.IsKVM() {
+		atexit.ExitWithMessage(1, configureIPAddressFailure)
+		// related to issues with the driver (ip is retrieved from the lease)
+	}
+
+	/*
```

Comment:
Let's remove commented code.

Reply:
commented code is what we should be doing in future.

Reply:
@gbraad In that case lets keep the code in the respective issue. -1 for putting some code related to future. I did not have the context when you explained it to me.